app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,19 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+
+export const getGreeting = createAsyncThunk('getGreeting', async() => {
+    const API_URL = 'http://localhost:3000/api/messages';
```

Comment:
- [ ] When I test both apps together this error comes up:

![image](https://github.com/HunterX-7/Hello-World/assets/77069011/051968bf-c9d2-4f46-8451-3d8e3f1f6595)

This is very common when you have client-server interaction, it basically means that your React client is not allowed to interact with your Rails server. Luckily it has a very easy fix, you can use the [`Rack-cors`](https://github.com/cyu/rack-cors) gem to allow your client to communicate with your server. 🤓 🔧

Reply:
Thanks for your suggestions, the changes that you required have been made and everything seems to be running good with the rack cors gem. 😄 

![Captura1](https://github.com/HunterX-7/Hello-World/assets/90111138/fc109c99-9e12-4039-bcea-8e9db24155ac)